EnglishFilter/sections/antiadblock.txt

```diff
@@ -729,6 +729,8 @@ smallseotools.com#@#.adsbygoogle
 ! https://github.com/AdguardTeam/AdguardFilters/issues/125135
 ansedo.com#%#//scriptlet("set-constant", "canRunAdvertBanner", "true")
 @@||ansedo.com/js/advert-banner.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/124880
+bbw-chan.nl##div[id^="justall"]
```

Comment:
Move it to Annoyances filter. I don't see that it affects the functionality of the site.